### ISSUE TITLE
Windows User Name With Spaces

### ISSUE THREAD

**Issue by KNTUM** [protz/ocaml-installer#49]
Hello,

unfortunately I first had a windows user name containing Spaces. Therefore often files and directories could not been found.
So I used the mkpasswd and modified it in order to change the Cygwin user. And I also changed the User name of my windows account. Now it seems that the installation worked but at the end of each insatllation or update command there comes a message that the \tmp\opam-_old user name with spaces_-Number cannot be removed. 

From where does the system get the old user? Is there a way to omit the problem?

**Comment by msprotz** [comment]
Hmm... I don't know about it.

I basically gave up on installation paths for OCaml containing spaces: most of the build scripts and Makefiles are incapable of dealing with paths that contain spaces (even on Unix).

**Comment by KNTUM** [comment]
I do not want to install the OCaml on paths with spaces. The home directory and everything now is complete without spaces.
The question I have is where does Cygwin or the Make Files get the old user name with spaces and ist there a way to change this. Normally I thought it would be fine if it change the user name of the Cygwin user. Unfortunately this did not help with this temp directories.

**Comment by msprotz** [comment]
Yep, I don't know about that. This sounds more like a cygwin-specific question, and my cygwin expertise, while greater than I would hope for, remains limited.
